### PR TITLE
{lib}[intel/2015b] phonopy 1.9.6 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description = """FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and
+ portable while capable of producing high-quality output (glyph images). It can be used in graphics libraries, display
+ servers, font conversion tools, text image generation tools, and many other products as well."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('libpng', '1.6.16')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb]: .18 ?

File: easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name = 'freetype'
+version = '2.5.5'
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb]: 2.6?

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb]: I need this because of the older matplotlib version, which is using 1.6.16

Comment (boegel) [inline easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb]: well, not older (it's the most recent matplotlib), it just hasn't gotten an update in a while (so this is just a toolchain update w.r.t. matplotlib)

File: easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/f/freetype/freetype-2.5.5-intel-2015b.eb]: required for the matplotlib 1.4.3 easyconfgi

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3fb4396edb89b8199859 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e91740878e8da7633abc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**